dockers/docker-ptf/Dockerfile

```diff
@@ -35,6 +35,7 @@ RUN sed --in-place 's/httpredir.debian.org/debian-archive.trafficmanager.net/' /
         python-dev          \
         wget                \
         cmake               \
+	isc-dhcp-server	    \
```

Comment:
I think it is better to remove this package from ptf docker since ptf docker is generic docker for testing purpose, what if some other one want to use ptf docker to test another dhcp server implementation? 

It is better to move this into the ansible playbook of the test itself.

---
In reply to: [84995018](https://github.com/Azure/sonic-buildimage/pull/38#discussion_r84995018) [](ancestors = 84995018)